extensions/amp-inline-gallery/1.0/amp-inline-gallery.css

```diff
@@ -1,3 +1,5 @@
+@import url(./amp-inline-gallery-pagination.css);
```

Comment:
Ditto. Please do this in a diff PR?

Reply:
https://github.com/ampproject/amphtml/pull/37567